src/pages/SignUpPage.scss

```diff
@@ -9,3 +9,24 @@
     top: 28.5px;
 
 }
+
+.signInLink {
+
```

Comment:
Remove empty line

Reply:
I just checked and there was supposed to be code there. Seems I did not stash the changes correctly. Will push it with the changes mentioned.